docs/component/random-sequence.md

```diff
@@ -0,0 +1,23 @@
+<!--
+    This markdown file was generated using `docs/documenter.php`.
+
+    Any edits to it will likely be lost.
+-->
+
+[*index](./../README.md)
+
+---
+
+### `Psl\RandomSequence` Component
```

Comment:
Would it make sense to put it in the *random namespaces that are already available?

`Psl\SecureRandom\Sequence` and `Psl\PseudoRandom\Sequence\MT...`? 

Not sure where to place the interface in that case.

Reply:
Was thinking : since the secure random sequence is the same as `SecureRandom\int()`, we might not need to use the interface for secure random?

Reply:
you might need it if you want predictable results during testing ( you would use MT ), but you want secure random sequence in production ( think an online casino, which would *need* to use a secure sequence for generating stuff, since MT is predictable ), so have an interface + different implementations is the way to go there.

> Not sure where to place the interface in that case.

yea, that's the issue :)